tests/sitetitle_test.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+import pytest_selenium
+
+# not needed for using pytest-selenium, just for using selenium directly
+from selenium import webdriver
+
+PLANTTRACER_TITLE = 'Plant Tracer Plant Tracer' # TODO: this is not the title we want. Should just be 'Plant Tracer'
+#PLANTTRACER_TITLE = 'Plant Tracer' # this is the desired title
```

Comment:
Lets actually put in the test, and have it fail, and fix it as part of this PR...

Reply:
I'm happy to help with the fix.

Reply:
You fixed it, thanks. Resolving.